src/modules/dockerEvents.js

```diff
@@ -4,7 +4,7 @@ import deepMerge from '../utils/deepMerge';
 
 const NANOSECONDS = 1000000;
 const DEFAULT_OPTIONS = {
-    format: '{{json .}}'
+    format: '"{{json .}}"'
```

Comment:
Hello @md-nowy! I appreciate your effort to fix this issue. Unfortunately this is not that straight forward due to Docker discrepancies between Windows and MacOS/Unix OS's. The format must be conditional in order to work in all OS's.